proxy/config/ssl_server_name.config.default

```diff
@@ -0,0 +1,36 @@
+--[[
+ssl_servername.config.default
+This configuration file
+    - sets the SSL actions to be performed based on the servername provided during SSL handhshake phase (SNI extension)
+    - sets the SSL properties required to make SSL connection with the next hop or origin server.
+
+Lua based Configuration file
+ Format : 
+ Actions available:
+   disable_h2 - removes H2 from the protocol list advertised by ATS; parameter required = None
+   TS_VERIFY_CLIENT - sets the verification flag for verifying the client certificate; parameters = Integer [0-2]
+   verify_origin_server - sets the verification flag for verifying the server certificate; parameters = Integer [0-2]
+   client_cert - sets the client certificate to present to the server specified in dest_host; parameters = certificate file .
+                     The location of the certificate file is relative to proxy.config.ssl.server.cert.path directory.
+   tunnel_route  - sets the e2e tunnel route
+ Example:
+
+
+params for client verification
+        NONE, MODERATE, STRICT
+
+sni_config = {
+                {
+                    fqdn='one.com',
+                    disable_h2 = <TRUE|FALSE>,
```

Comment:
It'd be great if this is protocol list. e.g. `proto={h1}`, `proto={h1,h2}`.
Otherwise, when new protocol is defined, we need to add another flag for it.

Reply:
Yes, I think we'll clean that up when we get the TsLuaConfig more robust.